src/lib/utils/snd_utils.c

```diff
@@ -0,0 +1,140 @@
+/* snd_utils.c
+**
+** Copyright (c) 2019, The Linux Foundation. All rights reserved.
```

Comment:
Can we switch to SPDX identifiers?

Reply:
sure .. we can do that .. only thing is current files has not moved to SPDX identifiers. So followed the same.

Reply:
Cc: @perexg. I would say we could start using SPDX identifiers and add a mental note to change existing files later.